server/app/services/MessageKey.java

```diff
@@ -177,7 +177,11 @@ public enum MessageKey {
   ERROR_FIRST_NAME("label.errorFirstName"),
   ERROR_LAST_NAME("label.errorLastName"),
   ERROR_INTERNAL_SERVER_TITLE("error.internalServerTitle"),
+  ERROR_INTERNAL_SERVER_TITLE_V2("error.internalServerTitle.v2"),
+  ERROR_INTERNAL_SERVER_SUBTITLE("error.internalServerSubtitle"),
   ERROR_INTERNAL_SERVER_DESCRIPTION("error.internalServerDescription"),
+  ERROR_INTERNAL_SERVER_HOME_BUTTON("error.internalServerHomeButton"),
+  ERROR_ERROR_CODE("error.errorCode"),
```

Comment:
**nitpick:** Up to you but maybe call this ERROR_STATUS_CODE and error.statusCode? The repeat of error reads a little funny to me

Reply:
i like that much better. updated.